llvm/CMakeLists.txt

```diff
@@ -117,6 +117,13 @@ endif()
 # This allows an easy way of setting up a build directory for llvm and another
 # one for llvm+clang+... using the same sources.
 set(LLVM_ALL_PROJECTS "bolt;clang;clang-tools-extra;compiler-rt;cross-project-tests;libc;libclc;lld;lldb;mlir;openmp;polly;pstl")
+if (${CMAKE_SYSTEM_NAME} MATCHES "AIX")
+  # Disallow 'openmp' as a LLVM PROJECT on AIX as the supported way is to use
+  # LLVM_ENABLE_RUNTIMES.
+  list(REMOVE_ITEM LLVM_ALL_PROJECTS openmp)
+endif()
+
+# The flang project is not yet part of "all" projects (see C++ requirements)
```

Comment:
nit: this is a dup. I think this is a duplicate/copy-paste error:
```suggestion
```

Reply:
Thanks for the review. The dup line is removed.